edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/Microsoft.Azure.Devices.Edge.Hub.CloudProxy.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.19.0" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.19.0-richma" />
```

Comment:
I don't have much context, but just curious why are you changing this?

Reply:
nested Edge need SDK changes that are not released yet, so I had to create my own SDK version for the interim.